README.md

```diff
@@ -26,13 +40,24 @@ found easily.  The documentation has decent coverage of all methods, but it does
 always make sense to reproduce everything that is in the C-SDK documentation, so
 it is recommended that you review it before getting started.
 
+This module requires a running `newrelic-daemon`, in other words the service `newrelic-infra` will return an initialization like this:
```

Comment:
```suggestion
This module requires a running `newrelic-daemon`, otherwise the service `newrelic-infra` will return an initialization diagnostic like this:
```